x/finality/keeper/power_dist_change.go

```diff
@@ -320,6 +323,11 @@ func (k Keeper) ProcessAllPowerDistUpdateEvents(
 	// sort new finality providers in activeBTCDels to ensure determinism
```

Comment:
comment about a line in which we set `fp.IsSlashed = true`  i.e
```
		if isSlashed {
			fp.IsSlashed = true
			continue
		}
```
shouldn't we also set `fp.TotalBondedSats = 0` there ? this would be inline with previous logic in which we set
`fp.TotalBondedSats = 0` at the begininng and the loop and later do not add any delegations this fp so effectively leaving it with `0`

Reply:
we can, but that would not modify the `newDc` since we are calling `continue` if the fp is slashed